Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/AliasedEnumPropertyDrawer.cs

```diff
@@ -0,0 +1,72 @@
+#if UNITY_EDITOR
```

Comment:
![Screenshot 2024-09-13 at 11 09 14](https://github.com/user-attachments/assets/3fc87754-2d72-4119-a846-f2b9561ff459)
![Screenshot 2024-09-13 at 11 11 23](https://github.com/user-attachments/assets/6b477234-d04b-45e0-b162-5f723599c821)
I am afraid this is not doing what its supposed to.....

The following things need to be addressed:
- Aliased values should be part of the same entry when the inspector is displayed, e.g. "South / X / A"
- Drop down need to allow all values to be picked, e.g. either "South / X / A" or each one individually would be fine from my perspective.
- At the moment when selecting e.g. South, the value displayed is "North", so something with mapping is incorrect

Reply:
I'll go over it and make sure it behaves and displays correctly